src/everything.rs

```diff
@@ -454,6 +455,7 @@ impl Everything {
         s.spawn(|_| self.events_ck3.validate(self));
         s.spawn(|_| self.provinces_ck3.validate(self));
         s.spawn(|_| self.wars.validate(self));
+        s.spawn(|_| Climate::validate_all(&self.database, self));
```

Comment:
Isn't it validated below in `self.database.validate(self);`?

Reply:
I guess the name is confusing :) This call is for a general validation of the item type, instead of per item.

Reply:
Maybe just `validate_item_type`